MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -664,10 +664,11 @@ - (void)updateResultViewError:(NSError *)error
 
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->